src/dune/coq_stanza.ml

```diff
@@ -0,0 +1,164 @@
+open Import
+open Dune_lang.Decoder
+open Stanza_common
+
+module Coqpp = struct
```

Comment:
Great! Originally I could not introduce `Coq_stanza` due to deps problems, as I was depending on some stuff in `Dune_file`, glad to see this solved!